### DISCUSSION TITLE
Multi-tenancy support

### DISCUSSION THREAD

**Discussion by williank** [nginx-proxy/nginx-proxy#2313] [Q&A]
Hello team, 

nginx-proxy have Multi-tenancy support ?

Where I would like to have multiple tenants on the same container but with different domains.

**Comment by buchdag** [comment]
Hi @williank 

I'm not sure I understood right but multi tenancy is something your app / service either support or does not support. Tenancy for a given proxied app should not be the reverse proxy responsibility 🤔
